src/MainWindow.vala

```diff
@@ -55,7 +55,12 @@ public class Music.MainWindow : Gtk.ApplicationWindow {
             icon = new ThemedIcon ("playlist-queue")
         };
 
-        selection_model = new Gtk.SingleSelection (playback_manager.queue_liststore);
+        search_placeholder = new Granite.Placeholder ("") {
+            description = _("Try changing search terms"),
+            icon = new ThemedIcon ("edit-find-symbolic")
+        };
+
+        selection_model = new Gtk.SingleSelection (list_model);
```

Comment:
Is there any reason we don't make `search_bar.filter_model` a property and access to it directly here?

I have rarely seen where we use the `out` keyword in the constructor.